issuing/cardholders.go

```diff
@@ -0,0 +1,92 @@
+package issuing
+
+import (
+	"time"
+
+	"github.com/checkout/checkout-sdk-go/common"
+)
+
+const (
+	issuing     = "issuing"
+	cardholders = "cardholders"
+	cards       = "cards"
+)
```

Comment:
This should be in a common file with the other path constants. You can name the file **issuing.go**